aten/src/ATen/native/cuda/layer_norm_kernel.cu

```diff
@@ -33,7 +33,12 @@ namespace at::native {
 namespace {
 
 constexpr int kCUDANumThreads = 256;
+#ifdef USE_ROCM
+// C10_WARP_SIZE is not constexpr for host code.
+#define kWarpSize C10_WARP_SIZE
+#else
 constexpr unsigned int kWarpSize = C10_WARP_SIZE;
```

Comment:
@xinyazhang Is it correct to declare this as a `constexpr` when `C10_WARP_SIZE` is [not a constexpr anymore](https://github.com/ROCm/pytorch/pull/2328/files#diff-8b8e2531c9927f406bcab344a60870250199dfd4909315296b7de13f3cb5d281R339) for CUDA either?